src/Service/Nph/NphOrderService.php

```diff
@@ -597,6 +597,32 @@ public function saveSamplesModification(array $formData, string $type, NphOrder
         return $status;
     }
 
+    public function updateSampleModificationBulk(array $formData, string $type): bool
+    {
+        $status = true;
+        foreach ($formData as $sampleId => $checked) {
```

Comment:
You are iterating through form data which will contain other data key-value pairs like "reason," "otherText," etc., which you want to exclude. That's why it's throwing the error below.

![image](https://github.com/all-of-us/healthpro/assets/6041980/8393adf6-b093-496b-b766-56d5c5058c56)